doc/index.rst

```diff
@@ -147,7 +147,8 @@ License
    :caption: Appendix
    :hidden:
 
-   ChangeLog/index
+   Coverage Report ➚ <https://edaa-org.GitHub.io/pySVModel/coverage/>
```

Comment:
This should not need to be an external reference. `Coverage Report ➚ <coverage/index>` should work and it would removed the warning about the file not being included in any toctree, isn't it?

Reply:
Now fixed.